### PR TITLE
Example form

### PR DESCRIPTION
**Pull request by elie222**

### CODE REVIEW

**Comment by agamm** [comment]
Can you fix the main conflicts first?

File: examples/next/app/ProfileForm.tsx
Comment (agamm) [inline examples/next/app/ProfileForm.tsx:1]: Wouldn't you want to keep this in the snippets so the next project is clean?

File: examples/next/app/api/user/settings/route.ts
Comment (agamm) [inline examples/next/app/api/user/settings/route.ts:1]: x2 Wouldn't you want to keep this in the snippets so the next project is clean?

File: examples/next/app/layout.tsx [hunk 1]
```diff
@@ -16,7 +17,9 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>{children}
+        <Toaster />
```
Comment (agamm) [inline examples/next/app/layout.tsx]: Wouldn't you want to keep this in the snippets so the next project is clean?

**Review by agamm — COMMENTED** [review]
Main question:
> Wouldn't you want to keep this in the snippets so the next project is clean?

**Comment by elie222** [comment]
Address all 3 comments:

Depends. Right now it's an example of using the generator on a project that already has a basis:
* Shadcn
* Existing structure/files/patterns

I didn't commit what the generator creates right now. But I did commit the extra files you commented on which are similar.
Toaster especially, the generation doesn't fully work without it.

I'll remove my changes to the Next.js project, and add a generator for the Toaster thing tomorrow. Would be a good example for us to have too. And it's simple.

**Review by agamm — DISMISSED** [review]
Ok for now, maybe open an issue to clean some of the files.

ps Wouldn't we like to also add those ui components via aleph eventually?

**Comment by elie222** [comment]
> Ok for now, maybe open an issue to clean some of the files.

What do you mean? I removed the extra files you commented on.

> ps Wouldn't we like to also add those ui components via aleph eventually?

Not sure what differentiates a UI component from other components. To React it's all the same. The hardcoded script generates a `Form.tsx` component. That's a form of UI component. And I'm adding one now that will add `Toaster` to the layout too.

**Pull request merged**